src/lib.rs

```diff
@@ -353,8 +361,11 @@ fn handle_packet<H, W>(data: &[u8],
                 no_ack_mode = true;
                 Response::String("OK")
             }
-            Command::PingThread(_) => {
-                Response::String("E01")
+            Command::PingThread(thread_id) => {
+                match handler.ping_thread(thread_id) {
+                    Result::Ok(_) => Response::String("OK"),
```

Comment:
If this will crop up a lot maybe we just want `Response::Ok`.

Reply:
...also given my previous comment about implementing `Into`, we could then do something like:
```rust
impl Into<Response> for () {
  fn into(self) -> Response { Response::Ok }
}
```
to simplify handler methods that don't need a return type.

Reply:
Yeah, I think `Response::Ok` is a good idea.